branding/custom-footer.html

```diff
@@ -1,7 +1,7 @@
 <div id="custom-footer-border"></div>
 <div id="custom-footer">
      <div class="custom-footer-logo">
-          <a href="https://lib.berkeley.edu" target="_blank">
+          <a href="https://lib.berkeley.edu" target="_blank" aria-label="Go to University of California, Berkeley Library homepage">
```

Comment:
I think this is more properly a `title` instead of an `aria-label`.

```suggestion
          <a href="https://lib.berkeley.edu" target="_blank" title="Go to University of California, Berkeley Library homepage">
```